contrib/cmake/cmake/FindUSD.cmake

```diff
@@ -0,0 +1,63 @@
+# Simple module to find USD.
```

Comment:
I think we should add a license block to our USD files. Did you author this file from scratch or is it based on an existing FindUSD.cmake file? 

A comment block on what needs to be set and what output variables are set if the library is found would be useful.

Reply:
I believe I found this module without any comments on top from here:
https://github.com/LumaPictures/openvdb-render/blob/master/cmake/FindUSD.cmake
But modified it a bit.

Reply:
Can you update the file with the license from Luma please? I don't think we should merge this without mentioning it's modified file from them. @andrewkaufman is this over the top?